import_tracker/rest.py

```diff
@@ -76,34 +81,54 @@ def getImports(query=None, user=None, unique=False, limit=None, offset=None, sor
     return processCursor(cursor, user)
 
 
-def moveLeafFiles(folder, user, assetstore):
+def moveLeafFiles(folder, user, assetstore, progress, job):
+    job = Job().load(job['_id'], force=True)
+    if job['status'] == JobStatus.CANCELED:
```

Comment:
Besides checking for the job being canceled at the start of a folder, we want to check this before each file move (basically just before any call to `moveFileToAssetstore`).  Moving a whole folder can be very slow, so cancelling only per-folder makes for a long wait (e.g., I have a folder with 25 large files in it, and I cancel after 2, I want it to cancel soonish, not in many minutes after moving the other 23 files).